File: new-algorithm/src/javaalgo/implementation/boj9498.java
package javaalgo.implementation;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class boj9498 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int score = Integer.parseInt(br.readLine());

        if (score >= 90 && score <= 100) System.out.println("A");
        else if (score >= 80 && score < 90) System.out.println("B");
        else if (score >= 70 && score < 80) System.out.println("C");
        else if (score >= 60 && score < 70) System.out.println("D");
        else System.out.println("F");
    }
}

File: new-algorithm/src/kotlinalgo/implementation/boj9498.kt
package kotlinalgo.implementation

import java.io.BufferedReader
import java.io.InputStreamReader

fun main() = with(BufferedReader(InputStreamReader(System.`in`))) {
    val score: Int = Integer.parseInt(readLine())

    println(
        when (score) {
            in 90..100 -> "A"
            in 80 until  90 -> "B"
            in 70 until  80 -> "C"
            in 60 until  70 -> "D"
            else -> "F"
        }
    )
}